src/main/java/novamachina/exnihilosequentia/ExNihiloSequentia.java

```diff
@@ -28,6 +30,7 @@
     public ExNihiloSequentia() {
         logger.debug("Starting Ex Nihilo: Sequentia");
         ModLoadingContext.get().registerConfig(ModConfig.Type.COMMON, Config.COMMON_CONFIG);
+        Config.loadConfig(Config.COMMON_CONFIG, FMLPaths.CONFIGDIR.get().resolve(ExNihiloConstants.ModIds.EX_NIHILO_SEQUENTIA + "-common.toml"));
```

Comment:
What exactly is this for?

Reply:
The next line is creating the items with the default parameters and doesnt get its parameters from config. thats why i need to get the configs parameters before creating the items.